app/dashboard/template.hbs

```diff
@@ -152,7 +152,26 @@
                                             <div class='col-sm-12 text-center'>
                                                 <p>{{t 'dashboard.quicksearch.no_projects.line1'}}</p>
                                                 <p>{{t 'dashboard.quicksearch.no_projects.line2'}}</p>
-                                                <img src='/assets/images/dashboard/quicksearch-min.png' alt='{{t 'dashboard.quicksearch.no_projects.preview_alt'}}' class='img-responsive center-block'>
+
+                                                <div local-class='iframe-container'>
+                                                    <iframe
+                                                        local-class='osf-video'
+                                                        src='https://www.youtube.com/embed/dLEIhJESIQA'
```

Comment:
Need an updated video link from product.

Reply:
@adius What does this mean?